src/pages/IndexPage.vue

```diff
@@ -123,8 +129,50 @@ const tempData = ref({
   name: '',
   age: '',
 });
+
+onMounted(() => {
+  getData();
+});
+function getData() {
+  axios.get('https://dahua.metcfire.com.tw/api/CRUDTest/a').then((res) => {
+    console.log(res);
+    blockData.value = res.data;
+  });
+}
+function addData(name, age) {
+  axios
+    .post('https://dahua.metcfire.com.tw/api/CRUDTest', {
+      name,
+      age,
+    })
+    .then((res) => {
+      console.log(res);
+      getData();
+      resetForm();
+    });
+}
+function resetForm() {
+  tempData.value = { name: '', age: '' };
+}
 function handleClickOption(btn, data) {
-  // ...
+  console.log(btn, data);
+  if (btn.status === 'edit') {
+    axios
+      .patch('https://dahua.metcfire.com.tw/api/CRUDTest', {
+        name: data.name,
+        age: data.age,
+      })
+      .then((res) => {
+        console.log(res);
+      });
+  } else if (btn.status === 'delete') {
+    axios
+      .delete(`https://dahua.metcfire.com.tw/api/CRUDTest/${data.id}`, {})
+      .then((res) => {
+        console.log(res);
+        getData();
+      });
+  }
```

Comment:
把 delete 和 edit 的 function 抽出，讓 handleClickOption 單純做流程控制就好

```suggestion
  if (btn.status === 'edit') {
    handleEdit()
  } else if (btn.status === 'delete') {
    handleDelete()
  }
```